frontend/src/pages/agreements/StepCreateBudgetLines.jsx

```diff
@@ -32,20 +29,40 @@ import {
     setSelectedAgreementType,
     setSelectedProcurementShop,
     setSelectedProject,
+    setSelectedCan,
+    setEditBudgetLineAdded,
+    setBudgetLineAdded,
+    deleteBudgetLineAdded,
+    duplicateBudgetLineAdded,
+    editBudgetLineAdded,
 } from "../agreements/createAgreementSlice";
 import { postBudgetLineItems } from "../../api/postBudgetLineItems";
 
 export const StepCreateBudgetLines = ({ goBack, goToNext, wizardSteps }) => {
     const dispatch = useDispatch();
     const navigate = useNavigate();
-    const budgetLinesAdded = useSelector((state) => state.createBudgetLine.budget_lines_added);
+    const budgetLinesAdded = useSelector((state) => state.createAgreement.budget_lines_added);
     const selectedProcurementShop = useSelector((state) => state.createAgreement.selected_procurement_shop);
     const selectedResearchProject = useSelector((state) => state.createAgreement.selected_project);
     const selectedAgreement = useSelector((state) => state.createAgreement.agreement);
+    const selectedCan = useSelector((state) => state.createAgreement.selected_can);
+    const enteredDescription = useSelector((state) => state.createAgreement.entered_description);
+    const enteredAmount = useSelector((state) => state.createAgreement.entered_amount);
+    const enteredMonth = useSelector((state) => state.createAgreement.entered_month);
+    const enteredDay = useSelector((state) => state.createAgreement.entered_day);
+    const enteredYear = useSelector((state) => state.createAgreement.entered_year);
+    const enteredComments = useSelector((state) => state.createAgreement.entered_comments);
+    const isEditing = useSelector((state) => state.createAgreement.is_editing_budget_line);
+    const budgetLineBeingEdited = useSelector((state) => state.createAgreement.budget_line_being_edited);
+    let loggedInUser = useSelector((state) => state.auth.activeUser.full_name);
     const [isAlertActive, setIsAlertActive] = React.useState(false);
     const [alertProps, setAlertProps] = React.useState({});
     const [showModal, setShowModal] = React.useState(false);
     const [modalProps, setModalProps] = React.useState({});
+    // NOTE: set to logged in user to Sheila if no name is found
+    if (!loggedInUser) {
```

Comment:
I'm curious about this logic..  effectively you shouldn't even be getting here if there's no `loggedInUser` 🤔

Reply:
think I needed a way to set a default name in case we weren't provided a full_name. In this case Sheila. lolz